.github/workflows/consumer_contract_tests.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       - name: Output consumer contract as non-breaking base64 string
         id: encode-pact
         run: |
-          NON_BREAKING_B64=$(cat service/build/pacts/wsm-consumer-bpm-provider.json | base64 -w 0)
+          NON_BREAKING_B64=$(cat service/build/pacts/wsm-bpm.json | base64 -w 0)
```

Comment:
@blakery Sorry, this was a late miss following your initial review.  Just highlighting it here for you for visibility.